packages/fiori/src/NotificationListItem.ts

```diff
@@ -175,7 +85,90 @@ const metadata = {
  * @implements sap.ui.webc.fiori.INotificationListItem, sap.ui.webc.main.IListItem
  * @public
  */
+@customElement("ui5-li-notification")
+@languageAware
+
+@event("_press")
 class NotificationListItem extends NotificationListItemBase {
+	/**
+	* Defines if the <code>titleText</code> and <code>description</code> should wrap,
+	* they truncate by default.
+	*
+	* <br><br>
+	* <b>Note:</b> by default the <code>titleText</code> and <code>decription</code>,
+	* and a <code>ShowMore/Less</code> button would be displayed.
+	* @type {sap.ui.webc.main.types.WrappingType}
+	* @defaultvalue "None"
+	* @public
+	* @name sap.ui.webc.fiori.NotificationListItem.prototype.wrappingType
+	* @since 1.0.0-rc.15
+	*/
+	@property({ type: WrappingType, defaultValue: WrappingType.None })
+	wrappingType!: WrappingType;
+
+	/**
+	* Defines the state of the <code>titleText</code> and <code>description</code>,
+	* if less or more information is displayed.
+	* @private
+	*/
+	@property({ type: Boolean })
+	_showMorePressed!: boolean;
+
+	/**
+	* Defines the visibility of the <code>showMore</code> button.
+	* @private
+	*/
+	@property({ type: Boolean })
+	_showMore!: boolean;
+
+	/**
+	* Defines the avatar, displayed in the <code>ui5-li-notification</code>.
+	*
+	* <br><br>
+	* <b>Note:</b> Consider using the <code>ui5-avatar</code> to display icons, initials or images.
+	* <br>
+	* <b>Note:</b>In order to be complaint with the UX guidlines and for best experience,
+	* we recommend using avatars with 2rem X 2rem in size (32px X 32px). In case you are using the <code>ui5-avatar</code>
+	* you can set its <code>size</code> property to <code>XS</code> to get the required size - <code>&lt;ui5-avatar size="XS">&lt;/ui5-avatar></code>.
+	*
+	* @type {sap.ui.webc.main.IAvatar}
+	* @slot
+	* @public
+	* @name sap.ui.webc.fiori.NotificationListItem.prototype.avatar
+	*/
+	@slot()
+	avatar!: Array<HTMLElement>;
+
+	/**
+	* Defines the elements, displayed in the footer of the of the component.
+	* @type {HTMLElement[]}
+	* @slot
+	* @public
+	* @name sap.ui.webc.fiori.NotificationListItem.prototype.footnotes
+	*/
+	@slot({ type: HTMLElement, individualSlots: true })
+	footnotes!: Array<HTMLElement>;
+
+	/**
+	* Defines the content of the <code>ui5-li-notification</code>,
+	* usually a description of the notification.
+	*
+	* <br><br>
+	* <b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+	*
+	* @type {Node[]}
+	* @slot description
+	* @public
+	* @name sap.ui.webc.fiori.NotificationListItem.prototype.default
+	*/
+	@slot({ type: Node, "default": true })
+	description!: Array<Node>;
+
+	_titleTextOverflowHeight: number;
+	_individualSlot!: string;
```

Comment:
_individualSlot is not a field on the NotificationListItem, it's something set by the framework to the slotted elements (in this case for footnotes slot). Perhaps you have added this cast `(el as typeof this)._individualSlot` work, but the cast does not feel right, because of the same reason - _individualSlot is not a filed on the NotificationListItem.

Looking into the UI5Element.ts, where the same thing occurred - we used `	(child as Record<string, any>)._individualSlot`, you can use it as well, and maybe discuss it after all other comments are reflected.